### PR TITLE
Dom/falkensee

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: src/compile/data/index.ts [hunk 1]
```diff
@@ -29,4 +29,9 @@ export interface DataComponent {
    * For facets, we store the reference to the root node.
    */
   facetRoot?: FacetNode;
+
+  /**
+   * Parse properties passed down from ancestors.
+   */
+  modelParse: Dict<string>;
```
Comment (kanitw) [inline src/compile/data/index.ts]: modelParse is a pretty confusing name -- it gives me no clue what it means when I first read the code.   maybe `ancestorParse`?

**Comment by kanitw** [comment]
Also, I think writing test for this would only take your extra 5-10 mins so please do it.

File: src/compile/data/index.ts (hunk 1 above)
Comment (domoritz) [inline src/compile/data/index.ts]: Done

**Comment by domoritz** [comment]
Wrote a test

**Comment by kanitw** [comment]
Cool.  Feel free to merge when test finishes running.

**Pull request merged**